index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="css/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h4>Eyasu</h4>
+        <img class="nav-pic" src="img/union.png" alt="">

```

Comment:
- [ ] Please put your hamburger menu image inside the `<nav>` tag below for semantic reasons.
- [ ] Please add the correct image path for the image because now it's not appearing online it may work offline with you but online now it's not working.
[Hint]: this is the correct image path for this one: `src="./img/Union.png"`.